packages/web-app/.storybook/en.json

```diff
@@ -1009,9 +1011,14 @@
   "The Wikicaves association has signed partnership with clubs and federations which provide data and which are informed of all the actions carried out besides we  have routine programmes that  regularly  check the quality of the data": "The Wikicaves association has signed partnership with clubs and federations which provide data and which are informed of all the actions carried out. Besides we  have routine programmes that  regularly  check the quality of the data.",
   "theme": "theme",
   "Theme": "Theme",
-  "There is no caves related to this group currently": "There is no caves related to this group currently.",
+  "There is no caves related to this group currently.": "There is no caves related to this group currently.",
```

Comment:
`There is no caves related to this group currently.` n'existe plus il semblerait et devrait donc etre supprimé non ?

Reply:
Vous m'aviez demandé de rajouter un point à la fin de cette phrase. N'étant pas à l'origine de cela, je ne pense pas que ce soit à moi de le supprimer.

Reply:
Tu peux la supprimer STP ? Tu touches a ce fichier via ta PR autant faire le nettoyage nécessaire plutot que d'attendre plus tard ;)

Reply:
Etant donné que je ne sais pas qui s'en sert, je ne voulais pas faire de bêtise 🙂
Mais je vais la supprimer de ce pas !